src/tests/ftest/control/dmg_scale.py

```diff
@@ -0,0 +1,185 @@
+    "engine_pool_scrubber_prev_duration_max",
+    "engine_pool_scrubber_prev_duration_mean",
+    "engine_pool_scrubber_prev_duration_min",
+    "engine_pool_scrubber_prev_duration_stddev",
+    "engine_pool_scrubber_scrubber_started",
+    "engine_pool_scrubber_scrubs_completed",
+    "engine_pool_started_at",
+    "engine_pool_vos_aggregation_akey_deleted",
+    "engine_pool_vos_aggregation_akey_scanned",
+    "engine_pool_vos_aggregation_akey_skipped",
+    "engine_pool_vos_aggregation_csum_errors",
+    "engine_pool_vos_aggregation_deleted_ev",
+    "engine_pool_vos_aggregation_deleted_sv",
+    "engine_pool_vos_aggregation_dkey_deleted",
+    "engine_pool_vos_aggregation_dkey_scanned",
+    "engine_pool_vos_aggregation_dkey_skipped",
+    "engine_pool_vos_aggregation_epr_duration",
+    "engine_pool_vos_aggregation_epr_duration_max",
+    "engine_pool_vos_aggregation_epr_duration_mean",
+    "engine_pool_vos_aggregation_epr_duration_min",
+    "engine_pool_vos_aggregation_epr_duration_stddev",
+    "engine_pool_vos_aggregation_merged_recs",
+    "engine_pool_vos_aggregation_merged_size",
+    "engine_pool_vos_aggregation_obj_deleted",
+    "engine_pool_vos_aggregation_obj_scanned",
+    "engine_pool_vos_aggregation_obj_skipped",
+    "engine_pool_vos_aggregation_uncommitted",
+    "engine_pool_vos_space_nvme_used",
+    "engine_pool_vos_space_scm_used",
+    "engine_pool_xferred_fetch",
+    "engine_pool_xferred_update",
+    "engine_pool_EC_update_full_stripe",
+    "engine_pool_EC_update_partial",
+    "engine_pool_block_allocator_alloc_hint",
+    "engine_pool_block_allocator_alloc_large",
+    "engine_pool_block_allocator_alloc_small",
+    "engine_pool_block_allocator_frags_aging",
+    "engine_pool_block_allocator_frags_large",
+    "engine_pool_block_allocator_frags_small",
+    "engine_pool_block_allocator_free_blks",
+    "engine_pool_ops_key2anchor"
+]
+
+
+class DmgScale(TestWithServers):
+    """Verify dmg commands works as expected in a large scale system.
+
+    :avocado: recursive
+    """
+
+    def test_dmg_scale(self):
+        """Run the following steps at Aurora and manually collect duration for each step.
+
+        0. Format storage
+        1. System query
+        2. Create a 100% pool that spans all engines
+        3. Pool query
+        4. Pool destroy
+        5. Create 49 pools spanning all the engines with each pool using a 1/50th of the capacity
+        6. Pool list
+        7. Query around 80 pool metrics
+        8. Destroy all 49 pools
+        9. System stop
+        10. System start
+
+        Jira ID: DAOS-10508.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium
+        :avocado: tags=deployment
+        :avocado: tags=DmgScale,test_dmg_scale
+        """
+        # This is a manual test and we need to find the durations from job.log, so add "##" to make
+        # it easy to search. The log is usually over 1 million lines.
+        self.log_step("## System query")
```

Comment:
Better to not put formatting for log_step because it already formats the messages
```suggestion
        self.log_step("System query")
```

Reply:
This is printed as:
```==> Step 4: ## System query [elapsed since last step: 0.00s]```
If we don't use `##`, We could search with `==>`, but I'm using `##` in other places such as total pool create duration. In my experience, it's easier to search with the same search string across the entire job.log than switching the strings to search different values.

Reply:
Thoughts @phender ? Similar rationale has come up before

Reply:
I think the cleaner way is to measure the duration for each step, but now we use the harness for some operations such as  `self.server_managers[0].system_stop()`,  so measuring the command duration isn't straightforward. Also, this test is manually executed only at RC (4 times in each RC), so I'm not sure if I want to put more effort into it.